### PR TITLE
Site-wide ToC v.2. - grouping items

### LINKED ISSUE #408: Site-wide Table of Contents v.2.
## Acceptance

- [ ] displays all pages grouped by content subfolders

## Notes

- only one level grouping?
  - i.e. group by content/<folder> and include files in subfolders of <folder> in this group as well

### PR DESCRIPTION
**Pull request by olayway**
Closes #408 

## Changes
- grouping of ToC items by first level subfolders of content folder

### CODE REVIEW

File: site/content/docs/site-wide-toc.md [hunk 1]
```diff
@@ -4,9 +4,7 @@ title: Site-wide table of contents (LHS Sidebar)
 
 Flowershow supports adding a site-wide table of contents in a LFH sidebar to allow your users to easily navigate to other pages on your website.
 
-![[site-wide-toc.png]]
-
-## Enabling / disabling the sidebar
+![[site-toc.png]]## Enabling / disabling the sidebar
```
Comment (khalilcodes) [inline site/content/docs/site-wide-toc.md:7]: minor fix

File: packages/template/pages/_app.tsx [hunk 2]
```diff
@@ -25,16 +25,17 @@ function MyApp({ Component, pageProps }) {
   const showComments =
     pageProps.showComments ??
     siteConfig?.comments?.pages?.includes(pageProps._raw?.sourceFileDir) ??
-    true;
+    false;
```
Comment (khalilcodes) [inline packages/template/pages/_app.tsx:28]: is the comments config error resolved here ?

**Review by khalilcodes — APPROVED** [review]

**Comment by khalilcodes** [comment]
@olayway 

not sure if we want the subscribed page in the list of toc ? 🤔 Should we have a way to handle the one's we don't need ? Something for later probably though.

Screen Shot 2023-02-02 at 11 39 44 PM

**Comment by olayway** [comment]
> @olayway
> 
> not sure if we want the subscribed page in the list of toc ? 🤔 Should we have a way to handle the one's we don't need ? Something for later probably though.
> 
> Screen Shot 2023-02-02 at 11 39 44 PM

@khalilcodes I don't think we're enabling the sidebar on our website atm anyway ;) But yes, this is sth we definitely want to allow for in the future.

**Pull request merged**